app/controllers/booking_requests_controller.rb

```diff
@@ -2,30 +2,48 @@ class BookingRequestsController < ApplicationController
   helper FormElementsHelper
 
   def index
-    processor = StepsProcessor.new(params, I18n.locale)
     @steps = processor.steps
     @step_name = processor.step_name
-    Instrumentation.append_to_log booking_step_rendered: processor.step_name
+
+    instrument_booking_request(step_name: @step_name)
 
     respond_to do |format|
       format.html { render processor.step_name }
     end
   end
 
   def create
```

Comment:
We agreed to disagree on this refactor (both pulling out assigning processor and the instrumentation). In my view this is unnecessary abstraction, which makes the code more complex.